README.md

```diff
@@ -4,7 +4,7 @@ This repository contains CSV files of Ontario COVID-19 assessment centre locatio
 
 ## Repository Structure
 
-Datasets are organized in directory by ownership and provenance.
+Datasets are organized in directory by ownership and provenance. All datasets have their publication dates attached in filenames.
 
 * `/ODS` directory: contains all versions of COVID-19 assessment centre locations fetched from [Ontario Open Data Service](https://data.ontario.ca/dataset/covid-19-assessment-centre-locations).
```

Comment:
We're mixing up a couple of groups here. The Ontario Digital Service is the government's tech arm, while the Ontario Data Catalogue is the open data repository maintained by the government. So the ODC is an ODS project. We've been calling the datasets founds on the ODC "ODS datasets" because they are collected by a team within the ODS. I suggest we rename the directory to "ODC" and change this line to reference the Ontario Data Catalogue.

Reply:
Good suggestion. It has been addressed and ready for another look. Thanks.